README.md

```diff
@@ -156,12 +156,17 @@ request state:
   consult CI logs to determine what happened. The bot does not attempt
   to merge this PR again until the PR branch or PR target branch change.
   When the bot notices that change, it removes this label.
+* `M-cleared-for-merge`: A human allowed the bot running in
+  `config::staged_run` mode to merge the PR. The label has no effect
+  unless the bot is running in that mode. The bot never sets this label.
+  The bot removes this label after successfully merging the PR.
```

Comment:
This looks correct to me, but somebody else could have questions: 
* What are differences between 'staged_run' and the label applied and their absence?
* Does the bot only merge the labeled PR (as described) or also other, cleanup steps, completing the PR?

To avoid them, I would rephrase this, emphasizing the fact that the label effectively cancels `staged_run` option (see the similar comment below).

Reply:
I could not quite grok your first bullet, but I agree with the second one -- I should not have used the overloaded "merge" term in this context. The current version avoids this problem.

Indirectly, this comment made me realize that the initial approach itself was wrong, thank you. Branch commit 378109e addresses that problem.